server/index.js

```diff
@@ -0,0 +1,56 @@
+require('dotenv/config');
+const express = require('express');
+
+const db = require('./database');
+const ClientError = require('./client-error');
+const staticMiddleware = require('./static-middleware');
+const sessionMiddleware = require('./session-middleware');
+
+const app = express();
+
+app.use(staticMiddleware);
+app.use(sessionMiddleware);
+
+app.use(express.json());
+
+app.get('/api/health-check', (req, res, next) => {
+  db.query('select \'successfully connected\' as "message"')
+    .then(result => res.json(result.rows[0]))
+    .catch(err => next(err));
+});
+
+app.get('/api/products', (req, res, next) => {
+  const sql = `
+  select
+  "name", "price", "image", "productId", "shortDescription"
+  from "products"
+  `;
+  db.query(sql)
+    .then(result => {
+      const products = result.rows;
+      res.status(200).json({
+        products
+      });
```

Comment:
Don't put `products` in curly brackets, you just want to send back the array of products. Should be:

```JAVASCRIPT
res.status(200).json(products)
```